docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md

```diff
@@ -54,6 +54,44 @@ Added a new function that returns initialization data passed when the add-in is
 
 ---
 
+### Mail signature
+
+#### [Office.context.mailbox.item.body.setSignatureAsync](/javascript/api/outlook/office.body?view=outlook-js-preview#setsignatureasync-data--options--callback-)
+
+Added a new function to the `Body` object that adds or replaces the signature in the item body in Compose mode.
+
+**Available in**: Outlook on Windows (connected to Office 365 subscription)
+
+#### [Office.context.mailbox.item.disableClientSignatureAsync](office.context.mailbox.item.md#methods)
+
+Added a new function that disables the client signature on the item in Compose mode.
```

Comment:
The signatures are not specific to the item, so could this be "Added a new function that disables the client signature for the sending account in Compose mode"

Reply:
will update.